### PR TITLE
Added middle steps to basic sell

### PR DESCRIPTION
**Pull request by piekczyk**
# [Added middle steps to basic sell](https://app.shortcut.com/oazo-apps/story/4516/ensure-the-middle-steps-in-the-basic-sell-form)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- added middle steps to basic sell form
  
## How to test 🧪
  <Please explain how to test your changes>
- use BasicBS feature flag, test it by adding / removing / updating trigger

### CODE REVIEW

File: features/automation/protection/controls/sidebar/SidebarSetupAutoSell.tsx [hunk 1]
```diff
@@ -1,161 +1,204 @@
...
-import { useObservable } from 'helpers/observableHook'
+import { SidebarFlow } from 'features/types/vaults/sidebarLabels'
 import { useUIChanges } from 'helpers/uiChangesHook'
 import { zero } from 'helpers/zero'
 import { useTranslation } from 'next-i18next'
-import React from 'react'
+import React, { useMemo } from 'react'
 import { Grid } from 'theme-ui'
 
 interface SidebarSetupAutoBuyProps {
```
Comment (johnnieskywalker) [inline features/automation/protection/controls/sidebar/SidebarSetupAutoSell.tsx:43]: SidebarSetupAutoSellProps

File: features/automation/protection/controls/sidebar/SidebarAuteSellCancelEditingStage.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+interface AutoSellInfoSectionControlProps {
+  cancelTxData: AutomationBotRemoveTriggerData
+  vault: Vault
+}
+
+function AutoSellInfoSectionControl({ cancelTxData, vault }: AutoSellInfoSectionControlProps) {
+  const { addGasEstimation$ } = useAppContext()
+
+  const addTriggerGasEstimationData$ = useMemo(() => {
```
Comment (johnnieskywalker) [inline features/automation/protection/controls/sidebar/SidebarAuteSellCancelEditingStage.tsx:23]: `removeTriggerGasEstimationData`

File: features/automation/protection/controls/sidebar/SidebarSetupAutoSell.tsx [hunk 3]
```diff
@@ -1,161 +1,204 @@
...
+    ? 'txInProgress'
+    : isFailureStage
+    ? 'txFailure'
+    : 'editing'
+
+  const isOwner = context.status === 'connected' && context.account === vault.controller
+  const isDisabled =
+    (isProgressStage ||
+      !isOwner ||
+      !isEditing ||
```
Comment (johnnieskywalker) [inline features/automation/protection/controls/sidebar/SidebarSetupAutoSell.tsx:131]: how about sth like `(isAddForm && !isEditing)` so button won't be disabled on remove form, when user would just switch to remove without editing anything

Comment (piekczyk) [reply features/automation/protection/controls/sidebar/SidebarSetupAutoSell.tsx:131]: its handled in isEditing constant

File: features/automation/protection/controls/sidebar/SidebarAuteSellCancelEditingStage.tsx (hunk 2 above)
Comment (piekczyk) [reply features/automation/protection/controls/sidebar/SidebarAuteSellCancelEditingStage.tsx:23]: done

File: features/automation/protection/controls/sidebar/SidebarSetupAutoSell.tsx (hunk 1 above)
Comment (piekczyk) [reply features/automation/protection/controls/sidebar/SidebarSetupAutoSell.tsx:43]: done

**Review by johnnieskywalker — APPROVED** [review]

**Pull request merged**